config/taxonomy/catalog.structs.schema.json

```diff
@@ -51,20 +55,42 @@
                                 "type": "string",
                                 "description": "name of column or component"
                             },
-                            "tags": { "$ref": "catalog.values.schema.json#/definitions/valid_metadata"}
+                            "tags": { "$ref": "#/definitions/tags" }
                         },
-                        "required":["name"]
+                        "required": ["name"]
                     }
                 }
             },
-            "required":["name"],
-            "additionalProperties": false
-        }
-    },
+            "required": ["name"],
+            "additionalProperties": {
+                "creationTime": {
+                    "type": "string",
+                    "description": "Time when the asset was created"
+                },
+                "CreatedBy": {
+                    "type": "string",
+                    "description": "Owner Id for the user who created the asset"
+                },
+                "updationTime": {
+                    "type": "string",
+                    "description": "Time when the dataset got updated"
+                },
+                "updatedBy": {
+                    "type": "string",
+                    "description": "Indicates who updted the asset"
+                },
+                "totalRatings": {
+                    "type": "string",
+                    "description": "Indicates the total rating of the asset"
```

Comment:
Are these ratings provided by the data users?  If so, please state.  Also, do we assume all catalogs will have this?

Reply:
These are metadata obtained from a particular asset. So, yes it is given by the data user as and when he is using the asset. This field is present in metadata obtained from both WKC and Egeria as well.

Reply:
Done